test/UnitTest/v1/data_ngsi10.py

```diff
@@ -1,3 +1,4 @@
+# Subscription request payload
```

Comment:
Make sure whether the subscription request payloads (for example, req given in subdata1, and others, that contains only entities and reference) are enough to subscribe for some entity.